src/Configuration/Settings.cs

```diff
@@ -123,6 +125,8 @@ public void Save()
                 settingsToSave.Password = string.Empty;
             }
 
+            settingsToSave.ProfilesDirectory = string.Empty;
+
```

Comment:
I don't think we want to save this.  Consider running the game in the following sequence

`classicuo.exe -profilesdirectory ~` // saves to home directory
followed by
`classicuo.exe -profilesdirectory` // should save to `pwd`, but will save to the recently specified `~` if this settings is saved